assets/src/edit-story/components/richText/formatters/letterSpacing.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { MULTIPLE_VALUE } from '../../form';
+import { NONE, LETTERSPACING } from '../customConstants';
+import {
+  togglePrefixStyle,
+  getPrefixStylesInSelection,
+} from '../styleManipulation';
+import { isStyle, numericToStyle, styleToNumeric } from './util';
+
+function letterSpacingToStyle(weight) {
+  return numericToStyle(LETTERSPACING, weight);
+}
+
+function styleToLetterSpacing(style) {
+  return styleToNumeric(LETTERSPACING, style);
+}
+
+function elementToStyle(element) {
+  const isSpan = element.tagName.toLowerCase() === 'span';
+  // This will implicitly strip any trailing unit from the value - it's assumed to be em
+  const letterSpacing = parseFloat(element.style.letterSpacing, 10);
```

Comment:
What's `10` here? Leftover from `parseInt`?

Reply:
🤦 (fixed in e2d2f236)